ci-operator/config/openshift/security-profiles-operator/openshift-security-profiles-operator-main.yaml

```diff
@@ -0,0 +1,65 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+canonical_go_repository: sigs.k8s.io/security-profiles-operator
+images:
+- dockerfile_path: Dockerfile.ubi
+  to: security-profiles-operator
+promotion:
+  name: "4.12"
+  namespace: ocp
+releases:
+  initial:
+    integration:
+      name: "4.12"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.12"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: APPARMOR_ENABLED=0 BPF_ENABLED=0 make test-unit
+  container:
+    from: src
+- as: e2e-operator
+  steps:
+    cluster_profile: aws
+    test:
+    - as: e2e-operator
+      cli: latest
+      commands: |
+        export E2E_CLUSTER_TYPE=openshift
+        export E2E_SPO_IMAGE=$IMAGE_FROM_CI
+        export E2E_SKIP_BUILD_IMAGES=true
+        export E2E_SKIP_NAMESPACED_TESTS=true
+        export E2E_TEST_SECCOMP=true
+        export E2E_TEST_PROFILE_RECORDING=false
+        export E2E_TEST_BPF_RECORDER=false
+        export E2E_TEST_LOG_ENRICHER=true
+        #export E2E_TEST_SELINUX=true
```

Comment:
Should we remove this?

Reply:
I should have added a comment (and I can), but I want to enable the tests again later (https://issues.redhat.com/browse/CMP-1468), but at the same time would prefer to merge the e2e config soon so that we can iterate on the tests outside the release repo, in the SPO openshift repo.

Reply:
so to answer more directly - I commented out the line instead of removing so that the comment keeps reminding me to re-enable the tests. I can remove the line or add a comment if that would look cleaner to you.

Reply:
Maybe a fixme to note what we need to do to re-enable this, but we can do that in a follow up, too.

Reply:
Done!

Reply:
Thanks!